Modules/Quotas/src/Quotas.Application/IntegrationEvents/Incoming/TierOfIdentityChanged/TierOfIdentityChangedIntegrationEventHandler.cs

```diff
@@ -21,7 +21,7 @@ public async Task Handle(TierOfIdentityChangedIntegrationEvent @event)
         var identity = await _identitiesRepository.Find(@event.IdentityAddress, CancellationToken.None, track: true);
         var newTier = await _tiersRepository.Find(@event.NewTier, CancellationToken.None, track: true);
 
-        await identity.ChangeTier(newTier, _metricCalculatorFactory, CancellationToken.None);
+        await identity!.ChangeTier(newTier, _metricCalculatorFactory, CancellationToken.None);
```

Comment:
I don't think this should be suppressed, see how `null` return value is handled in `Backbone.Modules.Quotas.Application.Identities.Commands.DeleteQuotaForIdentity.Handler` - I think we need some kind of reaction to the `null` returned.

Reply:
👍